conduit/plugins/importers/algod/metrics.go

```diff
@@ -2,17 +2,5 @@ package algodimporter
 
 import "github.com/prometheus/client_golang/prometheus"
 
-// Prometheus metric names
-const (
-	GetAlgodRawBlockTimeName = "get_algod_raw_block_time_sec"
-)
-
-// Initialize the prometheus objects
-var (
-	GetAlgodRawBlockTimeSeconds = prometheus.NewSummary(
-		prometheus.SummaryOpts{
-			Subsystem: "indexer_daemon",
-			Name:      GetAlgodRawBlockTimeName,
-			Help:      "Total response time from Algod's raw block endpoint in seconds.",
-		})
-)
+// getAlgodRawBlockTimeSeconds is used to record how long it took to fetch the block.
+var getAlgodRawBlockTimeSeconds prometheus.Summary
```

Comment:
it looks like GetAlgodRawBlockTimeSeconds can be removed from [util/metrics/metrics.go](https://github.com/algorand/indexer/blob/develop/util/metrics/metrics.go#L18)

Reply:
It's used by fetcher, but maybe fetcher is completely deprecated now?

Reply:
Added a note to remove this metric when fetcher is removed: https://github.com/algorand/indexer/issues/1347